Automattic/wp-calypso
# Remove UpsellSwitch HOC (no longer used)

#### Changes proposed in this Pull Request

* Remove the `UpsellSwitch` component, since it's no longer referenced anywhere.
* Move the `UpsellComponentProps` type declaration into the file for `BackupsUpsellPage` (i.e., the only remaining place this type is referenced).

#### Testing instructions

1. **Inspect the code:** Search through the `wp-calypso` codebase and verify the `UpsellSwitch` component is no longer referenced.
2. **Regression testing:** Visit the Backup and Scan pages, selecting as many different types of site as you can (multisite, Atomic, self-hosted, etc.). Verify up-sell prompts show up on these pages when they're supposed to, and don't show up when they're not.
